src/components/moderation/modlogs.ts

```diff
@@ -22,10 +23,12 @@ function is_autoremove(info: moderation_edit_info) {
 }
 
 export default class Modlogs extends BotComponent {
-    constructor(wheatley: Wheatley) {
+    constructor(wheatley: Wheatley, commands: CommandSetBuilder) {
```

Comment:
Perhaps added in an early iteration?

Reply:
yeah, exactly that, I did at least three passes to make sure l fixed them all -.-